cmake/add_project_smqtk.cmake

```diff
@@ -12,26 +12,60 @@ set( VIAME_PROJECT_LIST ${VIAME_PROJECT_LIST} smqtk )
 if( WIN32 )
   message( FATAL_ERROR "SMQTK not yet supported on WIN32" )
 else()
-  if( VIAME_SYMLINK_PYTHON )
-    set( SMQTK_PIP_CMD
-      pip install --user -e .[postgres] )
-  else()
-    # This is only required for no symlink install without a -e with older
-    # versions of pip, for never versions the above command works with no -e
-    set( SMQTK_PIP_CMD
-      pip install --user file://${VIAME_PACKAGES_DIR}/smqtk\#egg=smqtk[postgres] )
-  endif()
+
+  # Logic for getting site packages is copied from kwiver
+  # (but it uses a new pycmd function that handles indentation -- ooohh. aahh.)
```

Comment:
Less oohh and aahh?  Not sure how others feel about flavorful comments.

Reply:
consider it a minor easter egg for the git archives (unless we rebase and squash).